modin/core/storage_formats/pandas/utils.py

```diff
@@ -18,73 +18,36 @@
 import pandas
 
 
-def get_default_chunksize(length, num_splits):
+def compute_chunksize(index_len, num_splits, default_block_size=None):
```

Comment:
Minor rename, `index` is too overloaded in my opinion.
```suggestion
def compute_chunksize(axis_len, num_splits, default_block_size=None):
```